packages/file/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: file
-version: 6.0.0
+version: 6.1.0
```

Comment:
There's a version mismatch: The changelog said this would be 6.0.1

Reply:
oops, good catch